composer.json

```diff
@@ -26,6 +26,7 @@
         "guzzlehttp/promises": "^1.3",
         "guzzlehttp/psr7": "^1.6",
         "jean85/pretty-package-versions": "^1.2",
+        "ocramius/package-versions": "^1.8",
```

Comment:
Why are we adding this explicitly? We only interface with `jean85/pretty-package-versions` right?

Reply:
This change isn't really related to this PR, anyway its reason is that we use the `PackageVersions\Versions::VERSION` constant and thus we should reference its package in our `composer.json` to avoid breaking of our code if all the other dependencies stops requiring it